lib/db/mysql/patches/patch-005-006.sql

```diff
@@ -0,0 +1,20 @@
+-- Add refreshTokens
+
+CREATE TABLE refreshTokens (
+  token BINARY(32) PRIMARY KEY,
+  clientId BINARY(8) NOT NULL,
+  INDEX tokens_client_id(clientId),
+  FOREIGN KEY (clientId) REFERENCES clients(id) ON DELETE CASCADE,
+  userId BINARY(16) NOT NULL,
+  INDEX tokens_user_id(userId),
+  email VARCHAR(256) NOT NULL,
+  scope VARCHAR(256) NOT NULL,
+  createdAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+) ENGINE=InnoDB CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+
+-- At expiresAt column for access tokens
+
+ALTER TABLE tokens ADD COLUMN expiresAt TIMESTAMP;
```

Comment:
IIUC this will default to `NULL` for existing columns.  We should either to an `UPDATE tokens SET expiresAt =BLAH` as part of this migration, or ensure that the code checking these expiry times can handle `null` values.